.github/workflows/release.yml

```diff
@@ -6,37 +6,20 @@ on:
       - v*
 
 jobs:
+  build:
+    uses: silx-kit/jupyterlab-h5web/.github/workflows/build.yml@main
```

Comment:
Ah nice, didn't know you could do this -- but what's the difference with adding the same trigger as above to the build workflow? Oh maybe to run only the `main` job? Ah and you can make it run before the `release` job, makes sense. I got there 😂